projects/packages/videopress/src/client/admin/components/edit-video-details/style.module.scss

```
@@ -34,3 +34,20 @@
 .input {
 	margin-top: calc( var( --spacing-base ) * 3 ); // 24px
 }
+
+.back-link {
+	.icon {
+		width: 16px;
+		margin-right: 4px;
+	}
+
+	.link {
+		--gray-70: #3C434A !important;
+		margin-bottom:28px;
```

Let's use `--spacing-base`:

```suggestion
		margin-bottom: calc( var( --spacing-base ) * 3 );
```